configure.ac

```diff
@@ -85,7 +85,7 @@ AC_PATH_TOOL(STRIP, strip)
 AC_PATH_TOOL(GCOV, gcov)
 AC_PATH_PROG(LCOV, lcov)
 dnl Python 3.x is supported from 3.4 on (see https://github.com/bitcoin/bitcoin/issues/7893)
-AC_PATH_PROGS([PYTHON], [python3.7 python3.6 python3.5 python3.4 python3 python])
+AC_PATH_PROGS([PYTHON], [python3.4 python3 python])
```

Comment:
Could you reorder those, such that python3.4 is merely the first to look for? Unless you are certain that there is a python3 alias on all systems. (Otherwise it would fall back to `python`, which commonly means `python2`)

Reply:
Done. That indeed seems safer, and still works with PyEnv.